python/graphstorm/gconstruct/remap_result.py

```diff
@@ -49,8 +54,51 @@
 # id_maps to each worker process.
 id_maps = {}
 
+def write_data_parquet_file(data, file_prefix):
+    """ Write data into disk using parquet format.
+
+        Parameters
+        ----------
+        data: dict of numpy Arrays
+            Data to be written into disk.
+        file_prefix: str
+            File prefix. The output will be <file_prefix>.parquet.
+    """
+    output_fname = f"{file_prefix}.parquet"
+    write_data_parquet(data, output_fname)
+
+def write_data_csv_file(data, file_prefix, delimiter=","):
+    """ Write data into disk using csv format.
+
+        Multiple values for a field are specified with a semicolon (;) between values.
+
+        Example:
+
+        .. code::
+
+            nide, emb
+            0, 0.001;1.2000;0.736;
+
+        Parameters
+        ----------
+        data: dict of numpy Arrays
+            Data to be written into disk.
+        file_prefix: str
+            File prefix. The output will be <file_prefix>.parquet.
+    """
+    output_fname = f"{file_prefix}.csv"
+    csv_data = {}
+    for key, vals in data.items():
+        if len(vals.shape) == 1:
+            csv_data[key] = vals.tolist()
+        elif len(vals.shape) == 2:
+            csv_data[key] = [";".join([str(v) for v in val]) \
+                             for val in vals.tolist()]
+    data_frame = pd.DataFrame(data)
+    data_frame.to_csv(output_fname, index=False, sep=delimiter)
```

Comment:
does this only save the vector data or does it save the id column too?

Reply:
Yes.